client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -538,53 +539,56 @@ class WpcomChecklistComponent extends PureComponent {
 	renderEmailSetupTask = ( TaskComponent, baseProps, task ) => {
 		const { translate, siteSlug } = this.props;
 
-		const getStartProps = ( startingTask, clicked_element ) => {
-			return {
-				email_forwarding: startingTask.email_forwarding,
-				clicked_element,
-			};
-		};
-
-		const emailSetupProps = {
-			title: translate( 'Get email for your site' ),
-			bannerImageSrc: '/calypso/images/stats/tasks/email.svg', // TODO: get an actual svg for this
-			description: translate(
-				'Subscribe to G Suite to get a dedicated inbox, docs, and cloud storage.'
-			),
-			duration: translate( '%d minute', '%d minutes', { count: 5, args: [ 5 ] } ),
-			onClick: () => {
-				this.trackTaskStart( task, getStartProps( task, 'button' ) );
-				page( `/domains/manage/email/${ siteSlug }` );
-			},
-			onDismiss: this.handleTaskDismiss( task.id ),
-		};
-
-		const emailForwardingUsed = {
-			completedTitle: translate( 'You set up email forwarding for your site' ),
-			completedDescription: translate(
-				'Want a dedicated inbox, docs, and cloud storage? {{link}}Upgrade to G Suite!{{/link}}',
-				{
-					components: {
-						link: (
-							<a
-								onClick={ () => this.trackTaskStart( task, getStartProps( task, 'hyperlink' ) ) }
-								href={ `/domains/manage/email/${ siteSlug }` }
-							/>
-						),
-					},
-				}
-			),
-			completedButtonText: translate( 'Upgrade' ),
-		};
-		const emailForwardingNotUsed = {
-			completedTitle: translate( 'You set up email for your site' ),
-		};
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				title={ translate( 'Get email for your site' ) }
+				completedTitle={ translate( 'You set up email for your site' ) }
+				bannerImageSrc={ '/calypso/images/stats/tasks/email.svg' } // TODO: get an actual svg for this
+				description={ translate(
+					'Subscribe to G Suite to get a dedicated inbox, docs, and cloud storage.'
+				) }
+				duration={ translate( '%d minute', '%d minutes', { count: 5, args: [ 5 ] } ) }
+				onClick={ () => {
+					this.trackTaskStart( task );
+					page( `/domains/manage/email/${ siteSlug }` );
+				} }
+				onDismiss={ this.handleTaskDismiss( task.id ) }
+			/>
+		);
+	};
 
-		const emailForwardingProps = task.email_forwarding
-			? emailForwardingUsed
-			: emailForwardingNotUsed;
+	renderEmailForwardingUpgradedToGSuiteTask = ( TaskComponent, baseProps, task ) => {
```

Comment:
This title is misleading because of the `d` at the end of `Upgraded`

It should be `renderEmailForwardingUpgradeToGSuiteTask`

Reply:
It is supposed to have the `d`. All of the tasks are named using the past tense, e.g. `email_verified`, `site_created`, `address_picked`. So this task is called `email_forwarding_upgraded_to_gsuite`, and it is rendered by `renderEmailForwardingUpgradedToGSuiteTask()`.

Reply:
Oh I see! Thanks for the clarification.